docs/user/whats-new.asciidoc

```diff
@@ -1,128 +1,19 @@
 [[whats-new]]
 == What's new in {minor-version}
 
-Here are the highlights of what's new and improved in {minor-version}.
-For detailed information about this release,
-check out the <<release-notes-7.16.0, release notes>>.
+coming::[7.17.0]
 
-Other versions: {kibana-ref-all}/7.15/whats-new.html[7.15] | {kibana-ref-all}/7.14/whats-new.html[7.14] | {kibana-ref-all}/7.13/whats-new.html[7.13] | {kibana-ref-all}/7.12/whats-new.html[7.12] | {kibana-ref-all}/7.11/whats-new.html[7.11] |
-{kibana-ref-all}/7.10/whats-new.html[7.10] | {kibana-ref-all}/7.9/whats-new.html[7.9] | {kibana-ref-all}/7.8/whats-new.html[7.8] | {kibana-ref-all}/7.7/release-highlights-7.7.0.html[7.7] |
-{kibana-ref-all}/7.6/release-highlights-7.6.0.html[7.6] | {kibana-ref-all}/7.5/release-highlights-7.5.0.html[7.5] | {kibana-ref-all}/7.4/release-highlights-7.4.0.html[7.4] |
-{kibana-ref-all}/7.3/release-highlights-7.3.0.html[7.3] | {kibana-ref-all}/7.2/release-highlights-7.2.0.html[7.2] | {kibana-ref-all}/7.1/release-highlights-7.1.0.html[7.1] |
-{kibana-ref-all}/7.0/release-highlights-7.0.0.html[7.0]
-
-//NOTE: The notable-highlights tagged regions are re-used in the
-//Installation and Upgrade Guide
+{kib} {minor-version} is a compatibility release for the {minor-version} Elastic Stack
+and contains no major enhancements.
```

Comment:
```suggestion
and contains no major features.
```

Reply:
The release notes include enhancements, the What's new includes new and notable features.